testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/cli/TestRun.java

```diff
@@ -262,7 +257,6 @@ private Environment getEnvironment()
                 container
                         // the test jar is hundreds MB and file system bind is much more efficient
                         .withFileSystemBind(testJar.getPath(), "/docker/test.jar", READ_ONLY)
-                        .withFileSystemBind(cliJar.getPath(), "/docker/trino-cli", READ_ONLY)
```

Comment:
why it was mounted here? Is it used in tests?

Reply:
Yes. There are CLI tests that are using cli directly.